R/assess_downloads.R

```diff
@@ -19,15 +19,16 @@ attr(assess_downloads_1yr, "label") <- "number of downloads in the past year"
 
 #' @export
 assess_downloads_1yr.pkg_ref <- function(x, ...) {
-  downloads_1yr <- sum(x$downloads$count)
-  pkg_metric(downloads_1yr, class = "pkg_metric_downloads_1yr")
+  pkg_metric_eval(class = "pkg_metric_downloads_1yr", {
+    sum(x$downloads$count)
+  })
 }
 
 
 
 #' Defining an Assessment Scoring Function
 #' Score a package for the number of downloads in the past year
-#'
+#'regularized
```

Comment:
Whoops - looks like I had a stray cursor at some point and typed into this comment. Will revert.